### PR TITLE
Multiselect

### PR DESCRIPTION
**Pull request by roman-bytes**
👍

### CODE REVIEW

**Comment by schne324** [comment]
How about some multiselect tests?

File: index.js [hunk 1]
```diff
@@ -96,6 +101,10 @@ module.exports = class Combobox {
       this.freshSelection = true;
     });
 
+    this.input.addEventListener('focus', () => {
+      this.input.value = this.selected >= 2 ? this.config.selectionValue(this.selected) : '';
```
Comment (schne324) [inline index.js]: this should be `this.selected.length`. Also, correct me if I'm wrong, but I thought we wanted to clear the input's value on focus of the input when multiple items are selected?

File: index.js [hunk 2]
```diff
@@ -201,6 +219,20 @@ module.exports = class Combobox {
       } else {
         this.filter().openList();
       }
+
+      //Handles if there are no results found
+      let noResults = this.list.querySelector('.no-results-text');
+      if (this.config.noResultsText && !this.currentOpts.length) {
```
Comment (schne324) [inline index.js]: We can simplify this slightly:
```js
if (this.config.noResultsText && !this.currentOpts.length && !noResults) { // create the noResults element
  noResults = document.createElement('div');
  Classlist(noResults).add('no-results-text');
  noResults.innerHTML = this.config.noResultsText;
  this.list.appendChild(noResults);
} else if (noResults && this.currentOpts.length) {
  this.list.removeChild(noResults);
}
```

File: index.js [hunk 3]
```diff
@@ -264,23 +296,47 @@ module.exports = class Combobox {
...
+      }
+    } else {
+      // Single select stuff
+      this.selected = [currentOpt];
     }
 
-    currentOpt.classList.add(this.config.selectedClass);
-    this.selected = currentOpt;
+    // Taking care of adding / removing selected class
+    if (currentOpt.classList.contains(this.config.selectedClass)) {
```
Comment (schne324) [inline index.js]: for older browser support and consistency, use `Classlist(currentOpt).contains(this.config.selectedClass)`

File: index.js (hunk 1 above)
Comment (roman-bytes) [reply index.js]: Good catch and yes we want to clear the input, I had it backwards.

**Pull request merged**